modules/core/shared/src/main/scala/lucuma/core/enums/StepGuideState.scala

```diff
@@ -8,9 +8,9 @@ import lucuma.core.util.Enumerated
 /**
  * An enumeration whose value determines whether guiding is enabled for a particular step.
  */
-enum GuideState(val tag: String, val name: String, val description: String) derives Enumerated {
+enum StepGuideState(val tag: String, val name: String, val description: String) derives Enumerated {
```

Comment:
I renamed this as it really is a property for a `Step`, but now I see it changes `lucuma-schemas`. Should I revert?

Reply:
And there is also a `GuideState` on `navigate` which refers to the guiding state in the telescope

Reply:
I think I should leave this question to someone who is actually using this stuff...

Reply:
Does the `GuideState` on navigate take other values? If it takes the same values, maybe it can be converted to a `NewType` of this one? (And leave this one unchanged)

Reply:
```scala
case class GuideState(
  mountOffload: MountGuideOption,
  m1Guide:      M1GuideConfig,
  m2Guide:      M2GuideConfig
) derives Eq
```